assets/multiqc_config.yml

```diff
@@ -0,0 +1,13 @@
+report_comment: >
```

Comment:
I've never even looked at the multiqc config to be honest, the workflow outputs trimmed read stats and that's it. I'll look into generating some BAM stats or the like - just a little counterintuitive because what you're looking for are the unmapped reads for circular candidates. Guess this isn't the case for the Diff Exp module.